tests/python_test.py

```diff
@@ -281,9 +283,9 @@ def test_read_resource_owners_invalid_align_size(tmpdir, sanlock_daemon):
 
 @pytest.mark.parametrize("size,offset", [
     # Smallest offset.
-    (MIN_RES_SIZE, 0),
+    (MIN_RES_SIZE, long(0)),
     # Large offset.
-    (LARGE_FILE_SIZE, LARGE_FILE_SIZE - MIN_RES_SIZE),
+    (LARGE_FILE_SIZE, long(LARGE_FILE_SIZE - MIN_RES_SIZE)),
```

Comment:
Best keep one of the offset as int, since we test different apis in these tests.

Reply:
following discussion about PyInt/PyLong in prev patch we will abandon this patch.
Py3 is long by default, Py2 will stay with Ints as of no-regression action course.

Reply:
I have alternative patch for parsing intput:
https://github.com/nirs/sanlock/commit/d3b7f20589dcdfc895ab4d44d404d12fd7e978bf

If we parse input with "K", we have no problem to accept long values, so there is no problem to return long values.

What do you think?